pkg/app/app.go

```diff
@@ -1475,6 +1653,31 @@ func (a *App) template(r *Run, c TemplateConfigProvider) (bool, []error) {
 	return true, errs
 }
 
+func (a *App) test(r *Run, c TestConfigProvider) []error {
```

Comment:
This could have better left not moved so that i was a bit easier for me to review 😉 (You didn't do anything except moving it to app.go so that it is located alphabetically next to `sync`, right?